src/main/java/com/couchbase/lite/router/Router.java

```diff
@@ -1301,6 +1300,7 @@ public Status do_POST_Document_revs_diff(Database _db, String _docID, String _at
     public Status do_POST_Document_compact(Database _db, String _docID, String _attachmentName) {
         Status status = new Status(Status.OK);
         try {
+            // Make Database.compact() thread-safe
```

Comment:
This comment might not be correct if want to make calling to the database object thread-safe.